doc/site/static/wren.js

```diff
@@ -34,6 +36,21 @@ window.onload = function() {
       jar.updateCode(content);
     }
 
+    share.onclick = (e) => {
+      var code = jar.toString()
+      var compressed = LZString.compressToEncodedURIComponent(code)
+      var url = location.protocol + "//" + location.host + location.pathname + "?code=" + compressed
```

Comment:
Perhaps use a template literal here?

Reply:
The code pretty much uses ES5 so I keeped it so.